demo/meeting/src/components/MediaStatsList/Styled.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import styled from 'styled-components';
+
+export const StyledList = styled.dl`
+  font-size: 0.75rem;
+  display: flex;
+  flex-wrap: wrap;
+  width: 28.125rem;
+`;
+
+export const StyledItemMetricName = styled.dt`
+  line-height: 1.5rem;
+  margin-left: 1.5rem;
+  width: 30%;
+`;
+
+export const StyledItemMetricValue = styled.dd`
+  line-height: 1.5rem;
+  width: 30%;
+  margin-left: 0;
+`;
+
+export const StyledMediaMetricsWrapper = styled.div`
```

Comment:
Actually this comment is for styling this in general. Have we checked the mobile experience for this? Our meeting demo is responsive hence would be great to extend the same here.

Reply:
Good point. I will add a ticket to revisit this for mobile experience once we have a design settled.